jax/_src/numpy/fft.py

```diff
@@ -200,15 +200,16 @@ def ifftn(a: ArrayLike, s: Shape | None = None,
       supported.
 
   Returns:
-    An array containing the inverse discrete Fourier transform of ``a``.
+    An array containing the multidimensional inverse discrete Fourier transform
+    of ``a``.
 
   See also:
-    - :func:`jax.numpy.fft.fftn`: Computes the discrete Fourier transform of
-      multidimensional array.
-    - :func:`jax.numpy.fft.fft`: Computes the discrete Fourier transform of
-      one-dimensional array.
-    - :func:`jax.numpy.fft.ifft`: Computes the inverse of discrete Fourier transform
-      of one-dimensional array.
+    - :func:`jax.numpy.fft.fftn`: Computes multidimensional discrete Fourier
+      transform.
+    - :func:`jax.numpy.fft.fft`: Computes one-dimensional discrete Fourier
+      transform.
+    - :func:`jax.numpy.fft.ifft`: Computes one-dimensional inverse discrete
+      Fourier transform.
```

Comment:
These lines should also all say "a" or "the".

Reply:
Done